src/notifications/getting_started.md

```diff
@@ -0,0 +1,77 @@
+
+![Breez SDK - Liquid Notifications](../images/BreezSDK_Liquid_Notifications.png)
+
+### Push notification data
+
+When receiving POST request data on the webhook URL, the NDS should get the `token`, `platform` and optional `app_data` from the URL params. If necessary, convert the token to a token that can be sent to the push notification service. Then format the data to be sent to the push notification service, which includes converting any JSON data into a string format. 
+
+By default the Notification Plugin should receive the push notification data in the following format:
+
+```json
+{
+    "notification_type": "swap_updated",
+    "notification_payload": "{ \"id\": \"\", \"status\": \"\" }",
+    "app_data": ""
+}
+```
+The structure and fields of this data can be changed by [customising the push messages](custom_messages.md) handling in the Notification Plugin to reflect how your NDS sends this data over push notifications.
+
+## Use cases
+
+The Notification Plugin handles several use cases by default to automatically process push notifications sent via the NDS from when an SDK service calls the registered webhook. If your use case isn't covered by the Notification Plugin, you can extend the plugin to [handle custom notifications](custom_notifications.md).
+
+#### Updating a swap
+
+When making a payment via lightning or a Bitcoin onchain address, the SDK uses a swap service to convert between the Liquid sidechain and Bitcoin onchain or lightning. A swap service is used to monitor changes to the swap status and send push notifications to wake the device to progress the swap. The Notification Plugin when receiving this notification from the NDS will start the SDK and let the SDK complete the swap or progress it to the next state. 
+
+The `swap_updated` notification type will be received by the webhook in the following format:
+```json
+{
+    "template": "swap_updated",
+    "data": {  
+        "id": "",    // The hashed id of the swap
+        "status": "" // The latest swap status
+    }
+}
+```
+
+#### Handling LNURL pay requests
+
+Having the ability to process push notifications when the application is in the background or closed also opens up the ability to handle payment requests from a static LNURL or lightning address. To do this the application also needs to register a webhook with an [LNURL-pay service](/guide/lnurl_pay_service.md), then when the LNURL service receives a request on the static LNURL address, it will forward it via the NDS to the application. The Notification Plugin handles the two-step flow for fulfilling these requests.
```

Comment:
```suggestion
Having the ability to process push notifications when the application is in the background or closed also opens up the ability to handle payment requests from a static LNURL or Lightning address. To do this, the application also needs to register a webhook with an [LNURL-pay service](/guide/lnurl_pay_service.md), then when the LNURL service receives a request on the static LNURL address, it will forward it via the NDS to the application. The Notification Plugin handles the two-step flow for fulfilling these requests.
```